wiki/Community/en.md

```diff
@@ -4,10 +4,11 @@ This page lists ways for people from all around osu! to get in touch with each o
 
 ## Places
 
+- [cavoe's osu! event](/wiki/Community/cavoe's_osu!_event)
 - [Community Discord servers](/wiki/Community/Discord_servers)
 - [Forum](/wiki/Community/Forum)
 - [Internet Relay Chat](/wiki/Community/Internet_Relay_Chat) (IRC)
-- [osu! Discord server](/wiki/Community/Discord_servers#official)
+- [Discord servers](/wiki/Community/Discord_servers#official)
```

Comment:
Link you are trying to rename is to official osu! Discord server, plus we already link community ones above (line 8)

```suggestion
- [osu! Discord server](/wiki/Community/Discord_servers#official)
```

Reply:
Ok I didn't see the one above, however both links lead me to the same page, so should I just delete the "Community Discord servers" and stay with just "Discord servers"?

Reply:
@cl8n would like your opinion here

Reply:
that looks like a leftover mistake from when the "osu discord server" article was turned into a section of "discord servers"

Reply:
so like this

```suggestion
- [Discord servers](/wiki/Community/Discord_servers)
```